sklearn/cross_validation.py

```diff
@@ -1438,6 +1438,12 @@ def cross_val_score(estimator, X, y=None, scoring=None, cv=None, n_jobs=1,
     -------
     scores : array of float, shape=(len(list(cv)),)
         Array of scores of the estimator for each run of the cross validation.
+
+    See Also
+    ---------
+    :func:`sklearn.metrics.make_scorer`:
+        Make a scorer from a performance metric or loss function.
```

Comment:
Check out the syntax for the numpy docstring standard: in the see also section,  the "func" syntax is not used.